newChecklist.md

```diff
@@ -1,155 +0,0 @@
-1. [CODE STYLE] - Don't use one-letter variable naming. (It could be confusing for another developer, who reads your code. Also, it would really hard to find this variable in the code editor using search)
-
-BAD EXAMPLE:
-```jsx
-onChange={(e) => setVariable(e.target.value)}
```

Comment:
Use arrow function notation for anonymous functions. Replace `function(e)` with `(e) =>`.